hail/src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -306,6 +306,7 @@ class FunctionBuilder[F >: Null](val parameterTypeInfo: Array[MaybeGenericTypeIn
 
       cn.accept(cw)
       bytes = cw.toByteArray
+      CheckClassAdapter.verify(new ClassReader(bytes), false, new PrintWriter(sw1))
```

Comment:
this shouldn't slow things down, right?

Reply:
I took it out. accidentally left it in after debugging.